tests/integration_tests/test_ws_client.py

```diff
@@ -0,0 +1,140 @@
+"""Tests for the WS API Client."""
+import pytest
+
+from solana.publickey import PublicKey
+from solana.rpc.websocket import WebSocketClient
+
+from .utils import assert_valid_response
+
+RETEST_LIMIT = 10
+
+ID = {}
```

Comment:
What do we need a global `ID` dictionary here?

Reply:
To unsub with old id from some things if you previously subbed

Reply:
Should I make it into a class?

Reply:
I think it will be better to run those specific tests with a new websocket client

Reply:
Yeah. I think running with a new websocket client is a good idea.